src/blosc2/lazyexpr.py

```diff
@@ -1933,6 +2022,32 @@ def lazyexpr(expression, operands=None, out=None, where=None):
     out: :ref:`LazyExpr`
         A :ref:`LazyExpr` is returned.
 
+    Examples
+    --------
+    >>> import blosc2
+    >>> import numpy as np
+    >>> dtype = np.float64
+    >>> shape = [3, 3]
+    >>> size = shape[0] * shape[1]
+    >>> a = np.linspace(0, 5, num=size, dtype=dtype).reshape(shape)
+    >>> b = np.linspace(0, 5, num=size, dtype=dtype).reshape(shape)
+    >>> a1 = blosc2.asarray(a)
+    >>> a1[:]
+    [[0.    0.625 1.25 ]
+    [1.875 2.5   3.125]
+    [3.75  4.375 5.   ]]
+    >>> b1 = blosc2.asarray(b)
+    >>> expr = 'a1 * b1 + 2'
+    >>> operands = { 'a': a1, 'b': b1 }
+    >>> lazy_expr = blosc2.lazyexpr(expr, operands=operands)
+    >>> f"Lazy Expression Created: {lazy_expr}"
+    Lazy Expression Created: a1 * b1 + 2
+    >>> expr_ = a1 * b1 + 2
+    >>> result = expr_.eval()
+    >>> result[:]
```

Comment:
The above two line can be fused into one: `expr_[:]`.  Remember that a LazyExpr supports the `__getitem__` interface.